django-rgd/tests/test_checksumfile.py

```diff
@@ -37,6 +37,18 @@ def test_create_url():
     assert model.name
 
 
+@pytest.mark.django_db(transaction=True)
+def test_create_url_local_file():
+    model = ChecksumFile()
+    model.type = FileSourceType.URL
+    model.url = 'file://opt/django-project/README.md'
```

Comment:
_Technically_, this should have the leading slash.

```suggestion
    model.url = 'file:///opt/django-project/README.md'
```

> Note that when omitting host, the slash is not omitted (while "file:///foo.txt" is valid, "file://foo.txt" is not, although some interpreters manage to handle the latter).

[wikipedia](https://en.wikipedia.org/wiki/File_URI_scheme)

...but, I think the way this is implemented will take both. Also, not sure how important it is to have that exact at this stage, so I wouldn't block this PR.

Reply:
Ah 🤦🏻‍♂️  - you're absolutely right, good catch